### ISSUE TITLE
Funding Backdrop: gather different funding models and ideas

### ISSUE THREAD

**Issue by herbdool** [backdrop-ops/backdrop-community#2]
Let's have this issue for just gathering different funding models and ideas. And leave the discussion for another issue.

**Comment by herbdool** [comment]
Charging for infrastructure services.

1. Backdrop the organization could charge for new **addon services** likes automatic backups or antispam. 
2. It could charge for a **long-term security branch**, sort of like that CiviCRM does, for those who don't want to be on the bleeding edge. Though Backdrop might not be changing that fast that it's difficult to keep up, at the moment.
3. Backdrop could, controversially, even look at charging for the services it already offers: **installing and updating via the web UI**. People would still be able to download the modules and core manually, but, in this scenario, they would need to pay for an access key in order to download via the web. I think it's controversial only because we're used to getting it for free right now. Charging for it wouldn't violate free software/open source principles since the code is still freely available.

**Comment by ghost** [comment]
I like Elementary OS's model (copied from https://github.com/backdrop/backdrop-issues/issues/4767):

---

For those not familiar, here's how Elementary OS's "Pay what you want" system works:

![Peek 2020-11-20 22-34](https://user-images.githubusercontent.com/2385329/99795911-b062b800-2b80-11eb-9b2d-555f48a1363c.gif)

I personally wouldn't recommend copying this exactly for Backdrop, since their system tends to be an 'opt-out' payment system (i.e. you pay to download it unless you specifically enter $0). I'd prefer to use an 'opt-in' donation/tip system for Backdrop.

Also, this payment system that Elementary uses for downloading their OS is the same that they also use in their AppCenter:

![Elementary AppCenter](https://miro.medium.com/max/3912/1*3YBvnkvrB9rGm9RE5kp5eQ.png)

---

Basically, you can 'pay/donate' to download modules or Backdrop itself, or enter $0 to download for free.

**Comment by indigoxela** [comment]
I'm not sure it this directly fits in here: what about "Sponsored by" badges in module README?

Something like:

- This port has been sponsored by (logo and link)
- Development has been sponsored by (logo and link)

**Comment by yorkshire-pudding** [comment]
I also quite like the CiviCRM idea of paying for membership.  As stated on #3 , for non-US businesses, donations from the business to overseas non-profits (with a very small number of exceptions) are problematic -they are not eligible expenditure and they are not donations.

**Comment by philsward** [comment]
I think a "membership" should have some sort of benefit, even if it's small.

**Update Dashboard**
I like the idea of a centralized place to update multiple sites from one dashboard. I would say 5 or less is free and 6+ have tiers based on how many sites are added. 6-20 = pro, 20+ = premium for example.

Larger web firms would gladly pay for a service to be able to view all of their sites update statuses and with a click of a button, blow out the latest updates to all of their sites from this dashboard. Drupal 7 had several such 3rd-party services but I guarantee more people would trust this service if it came directly from Backdrop.

This could be extended to offer other information about multiple sites such as enabled modules, server details etc.

To stay fair with "open source", the Backdrop facing side could be added as a project where a firm could setup their own dashboard for free but they are required to set it up and maintain it. The paid version (accessed directly through b.org) gets the latest updates and whistles and bells first with the free open source version lagging behind 6 months or so.

**Comment by stpaultim** [comment]
This is being discussed in Zulip today. I have offered to help schedule and participate in a meeting of folks interested in this topic and willing to work on it. 

I am suggesting that we start with the idea of a Backdrop Business Association - which among other things might work to fund focused Backdrop development, but I'm open to the discussion GOING ANYWHERE. 

I am looking for folks with ideas AND a willingness to work on this. Solving this problem will require a lot of work. If anyone is interested in joining this effort, post here or contact me privately.

**Comment by yorkshire-pudding** [comment]
@stpaultim I have ideas and happy to discuss.  I agree that setting up an organisation could be a lot of work but is worth exploring.
